dds_web/api/project.py

```diff
@@ -188,6 +188,129 @@ def post(self):
 
         return {"message": return_message}
 
+    @auth.login_required(role=["Unit Admin", "Unit Personnel"])
+    @logging_bind_request
+    @json_required
+    @handle_validation_errors
+    @handle_db_error
+    def patch(self):
+        """Partially update a the project status"""
+        # Get project ID, project and verify access
+        project_id = dds_web.utils.get_required_item(obj=flask.request.args, req="project")
+        project = dds_web.utils.collect_project(project_id=project_id)
+        dds_web.utils.verify_project_access(project=project)
+
+        # Get json input from request
+        json_input = flask.request.get_json(silent=True)  # Already checked by json_required
+
+        # the status has changed at least two times,
+        # next time the project expires it wont change again -> error
+        if project.times_expired >= 2:
+            raise DDSArgumentError(
+                "Project availability limit: The maximum number of changes in data availability has been reached."
+            )
+
+        # Operation must be confirmed by the user - False by default
+        confirmed_operation = json_input.get("confirmed", False)
+        if not isinstance(confirmed_operation, bool):
+            raise DDSArgumentError(message="`confirmed` is a boolean value: True or False.")
+        if not confirmed_operation:
+            warning_message = "Operation must be confirmed before proceding."
+            # When not confirmed, return information about the project
+            project_info = ProjectInfo().get()
+            project_status = self.get()
+            json_returned = {
+                **project_info,
+                "project_status": project_status,
+                "warning": warning_message,
+                "default_unit_days": project.responsible_unit.days_in_expired,
```

Comment:
Missed this but this should be `days_in_available`
```suggestion
                "default_unit_days": project.responsible_unit.days_in_available,
```